src/styles/breakpoints.scss

```diff
@@ -0,0 +1,11 @@
+@import "~bootstrap/scss/_functions.scss";
+@import "~bootstrap/scss/_variables.scss";
+@import "~bootstrap/scss/mixins/_breakpoints.scss";
+
+@mixin mobile {
```

Comment:
Are we using `media-breakpoint-down` or these? I think we should use one or the other and probably delete this file?

Reply:
done. :)